package.json

```
@@ -2,9 +2,11 @@
 	"scripts": {
 		"build": "webpack",
 		"watch": "webpack --watch --env target=firefox",
-		"start": "concurrently \"npm:watch\" \"npm:web-ext:run:firefox\"",
+		"start": "npm run start:firefox",
+		"start:firefox": "concurrently \"npm:watch\" \"npm:web-ext:run:firefox\"",
```

I guess we need to extract the `--env target=firefox` into here, and change it to `chrome` for the `start:chrome` script.

(Suggestion not tested)

```suggestion
		"start:firefox": "concurrently \"npm:watch -- --env target=firefox\" \"npm:web-ext:run:firefox\"",
```